### PR TITLE
920569 updated the PdfLoadedWidgetAnnotation to access the widget annotation in the PDF document

### PR DESCRIPTION
**Pull request by sameerkhan001**
Hi All,

Task id:https://dev.azure.com/EssentialStudio/Document%20Processing%20Libraries/_workitems/edit/920569

Task name:Need to update the PdfLoadedWidgetAnnotation to access the widget annotation in the PDF document

Regards,
Sameerkhan N

### CODE REVIEW

File: Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+//Load the PDF document.
+FileStream docStream = new FileStream(Path.GetFullPath(@"Data/Input.pdf"), FileMode.Open, FileAccess.Read);
+PdfLoadedDocument loadedDocument = new PdfLoadedDocument(docStream);
+// Loop through each page in the loaded PDF document.
+foreach (PdfLoadedPage page in loadedDocument.Pages)
+{
+    // Access the form within the loaded document.
+    PdfLoadedForm form = loadedDocument.Form;
+    // Check if the forms are null.
+    if (form == null)
```
Comment (chinnumuniyappan) [inline Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs:14]: Whether it is required to check whether form is null. It is not needed. The unlinked annotation will be added to the collection. So remove this condition.

File: Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+﻿using Syncfusion.Pdf.Interactive;
+using Syncfusion.Pdf.Parsing;
+using Syncfusion.Pdf;
+
+//Load the PDF document.
+FileStream docStream = new FileStream(Path.GetFullPath(@"Data/Input.pdf"), FileMode.Open, FileAccess.Read);
```
Comment (chinnumuniyappan) [inline Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs:6]: Don't attach the custom documents?

**Review by chinnumuniyappan — CHANGES_REQUESTED** [review]

File: Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs (hunk 1 above)
Comment (sameerkhan001) [reply Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs]: Removed

File: Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs (hunk 2 above)
Comment (sameerkhan001) [reply Forms/Modify-the-widget-annotation-in-PDF-document/.NET/Modify-the-widget-annotation-in-PDF-document/Program.cs:6]: Changed input document

**Review by chinnumuniyappan — APPROVED** [review]

**Pull request merged**